project/ISSUE-TRIAGE.md

```diff
@@ -86,19 +66,25 @@ labels.
 | Level            | Experience level guideline                                                                                               |
 |------------------|--------------------------------------------------------------------------------------------------------------------------|
 | exp/beginner     | You have made less than 10 contributions in your life time to any open source project.                                   |
-| exp/novice       | You have made more than 10 contributions to an open source project or at least 5 contributions to Docker.                | 
-| exp/proficient   | You have made more than 5 contributions to Docker which amount to at least 200 code lines or 1000 documentation lines.   | 
-| exp/expert       | You have made less than 20 commits to Docker which amount to 500-1000 code lines or 1000-3000 documentation lines.       | 
-| exp/master       | You have made more than 20 commits to Docker and greater than 1000 code lines or 3000 documentation lines.               | 
+| exp/intermediate | You have made more than 5 contributions to Docker which amount to at least 200 code lines or 1000 documentation lines.   |
+| exp/expert       | You have made less than 20 commits to Docker which amount to 500-1000 code lines or 1000-3000 documentation lines.       |
```

Comment:
did you intend this to say `expert` == `less than 20 commits` ? if its intentional, then it sounds odd without adding a not saying that if there's no level tag, it could either be way freakishly hard, or unclassified :)